docs/item_set_names.md

```diff
@@ -1,38 +1,71 @@
-# item\_set\_names
+# item_set_names
 
 [<-Back-to:World](database-world)
 
-**The \`item\_set\_names\` table**
+**The \`item_set_names\` table**
 
 `table-no-description`
 
 **Table Structure**
 
-| Field              | Type         | Attributes | Key | Null | Default | Extra | Comment |
-| ------------------ | ------------ | ---------- | --- | ---- | ------- | ----- | ------- |
-| [entry][1]         | MEDIUMINT    | UNSIGNED   | PRI | NO   |         |       |         |
-| [name][2]          | VARCHAR(255) | SIGNED     |     | NO   |         |       |         |
-| [InventoryType][3] | TINYINT      | UNSIGNED   |     | NO   | 0       |       |         |
-| [WDBVerified][4]   | SMALLINT     | UNSIGNED   |     | NO   | 1       |       |         |
-
-[1]: #entry
-[2]: #name
-[3]: #inventorytype
-[4]: #wdbverified
+| Field                           | Type         | Attributes | Key | Null | Default | Extra | Comment |
+| ------------------------------- | ------------ | ---------- | --- | ---- | ------- | ----- | ------- |
+| [entry](#entry)                 | MEDIUMINT    | UNSIGNED   | PRI | NO   |         |       |         |
+| [name](#name)                   | VARCHAR(255) | SIGNED     |     | NO   |         |       |         |
+| [InventoryType](#inventorytype) | TINYINT      | UNSIGNED   |     | NO   | 0       |       |         |
+| [WDBVerified](#wdbverified)     | SMALLINT     | UNSIGNED   |     | NO   | 1       |       |         |
 
 **Description of the fields**
 
-### entry
+### Entry
 
-Entry from item\_template
+Item [Entry](item_template#entry) ID used in [item_template](item_template)
 
-### name
+### Name
 
-Name of the item
+Item [Name](item_template#name) used in [item_template](item_template)
 
 ### InventoryType
 
-`field-no-description|3`
+Which slot the Item will be equipped on.
+
+<details>
+
+<summary>Click to view</summary>
```

Comment:
I dont see the use for details

Reply:
While sure item_set_names is a small page, using `<details>` more often would save in a lot of page scroll.

From this 

![image](https://github.com/user-attachments/assets/a38c2925-4338-43da-80e3-fc7c237bb55a)


to

![image](https://github.com/user-attachments/assets/245db13f-7140-4c78-a2e1-1b29a887d7a1)

Reply:
I see why it could be nice. But I think it should be accessible at face value in a wiki

Reply:
When im back at pc I will remove it and make a push then

Reply:
Done